obspy/io/gse2/bulletin.py

```diff
@@ -0,0 +1,1071 @@
+
+        for i in range(2):
+            try:
+                mag_errors = magnitudes[i].mag_errors
+                mag_errors.uncertainty = float(magnitude_errors[i])
+            except (AttributeError, ValueError):
+                pass
+
+        # No match for 'g' (guess)
+        # We map 'g' to 'manual' and create a comment for origin
+        try:
+            origin.evaluation_mode = EVALUATION_MODES[analysis_type]
+
+            if analysis_type == 'g':
+                # comment: 'GSE2.0:antype=g'
+                text = 'GSE2.0:antype=g'
+                comment = self._comment(text)
+                origin.comments.append(comment)
+        except KeyError:
+            self._warn('Wrong analysis type')
+
+        if location_method not in LOCATION_METHODS.keys():
+            location_method = 'o'
+        method = LOCATION_METHODS[location_method]
+        method_id = "method/%s" % method
+        origin.method_id = self._get_res_id(method_id)
+
+        if event_type not in EVENT_TYPES.keys():
+            event_type = 'uk'
+            self._warn('Wrong or unknown event type')
+        event_data = EVENT_TYPES[event_type]
+        event.event_type_certainty, event.event_type = event_data
+
+        # comment: 'GSE2.0:evtype=<evtype>'
+        if event_type:
+            text = 'GSE2.0:evtype=%s' % event_type
+            comment = self._comment(text)
+            event.comments.append(comment)
+
+    def _parse_region_name(self, line, event):
+        event_description = EventDescription()
+        event_description.text = line.strip()
+        event_description.type = EventDescriptionType('region name')
+        event.event_descriptions.append(event_description)
+
+    def _parse_arrivals(self, event, origin, origin_res_id):
+        # Skip header of arrivals
+        next(self.lines)
+
+        # Stop the loop after 2 empty lines (according to the standard).
+        previous_line_empty = False
+
+        for line in self.lines:
+            line_empty = not line or line.isspace()
+
+            if not self.event_point_separator:
+                # Event are separated by two empty lines
+                if line_empty and previous_line_empty:
+                    break
+            else:
+                # Event are separated by '.'
+                if line.startswith('.'):
+                    break
+
+            previous_line_empty = line_empty
+
+            if line_empty:
+                # Skip empty lines when the loop should be stopped by
+                # point
+                continue
+
+            magnitude_types = []
+            magnitude_values = []
+
+            fields = self.fields['arrival']
+
+            station = line[fields['sta']].strip()
+            distance = line[fields['dist']].strip()
+            event_azimuth = line[fields['ev_az']].strip()
+            evaluation_mode = line[fields['picktype']].strip()
+            # direction = line[fields['direction']].strip()
```

Comment:
Indeed, I didn't know where to put them. I'm digging to see what I can do with these fields. In particular, I'm not sure about the `arrival azimuth` field (position 43 to 52). Do you know if it's equivalent to the `Pick.backazimuth` in obspy?